pombola/south_africa/templates/core/person_detail.html

```diff
@@ -204,6 +195,26 @@ <h3 class="mp-block__title">Contact Numbers:</h3>
               {% endfor %}
             {% endif %}
 
+            {% if person.has_ever_been_member_of_nat_prov_legislature and email_contacts %}
+              {% if not phone_contacts %}
+                <h3 class="mp-block__title">Contact Numbers:</h3>
```

Comment:
what does this have to do with contact numbers?

So... the block above only shows the contact numbers heading if there are phone contacts. And then this inserts the phone contacts heading (duplication) so that we show "contact numbers" if there are email addresses, even if there aren't phone numbers?

Why are we removing the email contacts section (heading and content) and sticking email addresses into the phone numbers and hacking an unrelated heading back in?

Reply:
moving this to https://openupsa.slack.com/archives/C3N7NPT09/p1642606976027700